### ISSUE TITLE
Extra blank slide in beamer

### ISSUE THREAD

**Issue by Jmuccigr** [olastor/paperify#6]
The following creates a blank slide after the title slide in beamer output, but not on my local pandoc installation:

```
---
title: My Presentation
author: Me!
---

# Heading

- one
- two
- three

-----

hi

-----

bye
```

**Comment by olastor** [comment]
Mhm, can't reproduce this in Firefox or Chrome (seems to work). Can you please recheck?

**Comment by Jmuccigr** [comment]
Yep, see the screenshot showing the blank page on Chromium. Same on Safari. Seems to be the leading header. If I delete it, the blank page goes away. If I do just the level 2 header, I get this error, but not with level 1 or 3 header:

```
An error occurred:

! Undefined control sequence.
<write> ...ARK [3][]{Outline0.0.1.3}{Hi}{\Parent2 
                                                  }% 1
l.77 \subsection{Hi}

pandoc: Error producing PDF
```

![screenshot 2018-02-20 11 20 01](https://user-images.githubusercontent.com/615115/36435606-1235204a-1630-11e8-80a1-11ca5dfe4712.png)

**Comment by olastor** [comment]
Sorry, I overlooked the `--to=beamer`. Yes, can reproduce it now.

**Comment by olastor** [comment]
Should be fixed via https://github.com/olastor/paperify-backend/commit/8494310bf5c824e60aab031d7c81f5125cb0cf71
